api/v1alpha1/types.go

```diff
@@ -16,11 +16,44 @@ limitations under the License.
 
 package v1alpha1
 
+import (
+	platformv1alpha1 "github.com/openshift/api/platform/v1alpha1"
+)
+
 var (
 	TypeApplied = "Applied"
 
 	ReasonSourceFailed    = "SourceFailed"
+	ReasonUnpackPending   = "ApplyPending"
 	ReasonApplyFailed     = "ApplyFailed"
 	ReasonApplySuccessful = "ApplySuccessful"
 	ReasonApplyPending    = "ApplyPending"
 )
+
+const (
+	SourcedBundleAnnotation = "platform.openshift.io/sourced-bundle"
+)
+
+func GetDesiredBundle(po *platformv1alpha1.PlatformOperator) string {
+	annotations := po.GetAnnotations()
+	desiredBundle, ok := annotations[SourcedBundleAnnotation]
+	if ok {
+		return desiredBundle
+	}
```

Comment:
nit: Any reason to not handle it this way?
```suggestion
	if desiredBundle, ok := annotations[SourcedBundleAnnotation]; ok {
		return desiredBundle
	}
```

Reply:
Ended up dropping this implementation in #41.